pubspec.yaml

```diff
@@ -30,6 +30,7 @@ dependencies:
   provider: ^6.0.5
   video_player: ^2.7.0
   visibility_detector: ^0.4.0
+  drag_select_grid_view: ^0.6.2
```

Comment:
Introducing such a package will significantly increase the risk to our functionality since we don't understand how it works and cannot control if it can serve us well.

Reply:
Thank you for the feedback. Yes, I think you are probably right. Introducing a new package brings uncertainties. And from my time working on this, making this package and drag_select_grid_view work together nicely was difficult, it would have been easier to create it from scratch. I tested https://github.com/fluttercandies/flutter_wechat_assets_picker/pull/660, it appears to work. Seems like a good solution.